community/io/src/test/java/org/neo4j/io/pagecache/impl/muninn/LargePageListIT.java

```diff
@@ -43,9 +45,9 @@ public void veryLargePageListsMustBeFullyAccessible() throws Exception
         long pageCacheSize = ByteUnit.gibiBytes( 513 ) + pageSize;
         int pages = Math.toIntExact( pageCacheSize / pageSize );
 
-        MemoryAllocator mman = MemoryAllocator.createAllocator( "2 GiB" );
+        MemoryAllocator mman = MemoryAllocator.createAllocator( "2 GiB", new LocalMemoryTracker() );
```

Comment:
This one is using a local tracker while the others in this tests are using the global instance. Looks suspicious.